tcti/tcti_device.c

```diff
@@ -234,3 +236,30 @@ TSS2_RC InitDeviceTcti (
 
     return rval;
 }
+
+TSS2_RC Tss2_Tcti_Device_Init (
+    TSS2_TCTI_CONTEXT *tctiContext,
+    size_t *size,
+    const char *conf
+    )
+{
+    const char *dev_path = conf != NULL ? conf : TCTI_DEVICE_DEFAULT;
+    TCTI_DEVICE_CONF dev_conf = {
+        .device_path = dev_path,
+    };
+
+    return InitDeviceTcti (tctiContext, size, &dev_conf);
+}
+
+const static TSS2_TCTI_INFO tss2_tcti_info = {
+    .name = "tcti-device",
+    .description = "TCTI module for communication with Linux kernel interface.",
+    .config_help = "Path to TPM character device: \"/dev/tpm0\" is default.",
```

Comment:
Instead of hardcoding /dev/tpm0, I think you should use TCTI_DEVICE_DEFAULT here:

```
.config_help = "Path to TPM character device: \""TCTI_DEVICE_DEFAULT"\" is default.",
```

Reply:
Makes perfect sense to me. For this to make sense `TCTI_DEVICE_DEFAULT` will need to be in a public header and currently it's only used internally. This is something that can go into $srcdir/include/tcti/tcti_device.h easy enough though.